django/tasks/backends.py

```diff
@@ -0,0 +1,119 @@
+from . import constants
+from .base import get_backend
+from .exceptions import ResultUnavailable
+
+
+class TaskResult(object):
+    """
+    Wrapper class representing a status of a task. Provides methods to retrieve
+    the task's status and result value/error.
+    """
+    def __init__(self, backend, task_id):
+        if not isinstance(backend, BaseBackend):
+            backend = get_backend(backend)
+        self._backend = backend
+        self.task_id = task_id
+
+    def __repr__(self):
+        return '<TaskResult %s - %r>' % (self.alias, self.task_id)
```

Comment:
From the python docs: "If at all possible, this should look like a valid Python expression that could be used to recreate an object with the same value (given an appropriate environment)." -- Seems easy enough in that case imo, unless we object to that in Django completely ;)